scriptworker/artifacts.py

```diff
@@ -232,18 +238,13 @@ def get_artifact_url(context, task_id, path):
         TaskClusterFailure: on failure.
 
     """
-    try:
-        url = unquote(context.queue.buildUrl('getLatestArtifact', task_id, path))
-    except AttributeError:
-        # taskcluster client 0.3.x
-        # XXX remove when we no longer want to support taskcluster<1.0.0
-        url = urljoin(
-            context.queue.options['baseUrl'],
-            'v1/' +
-            unquote(context.queue.makeRoute('getLatestArtifact', replDict={
-                'taskId': task_id,
-                'name': path
-            }))
+    if path.startswith("public/"):
+        url = context.queue.buildUrl('getLatestArtifact', task_id, path)
+    else:
+        url = context.queue.buildSignedUrl(
+            'getLatestArtifact', task_id, path,
+            # XXX Can set expiration kwarg in (int) seconds from now;
```

Comment:
Minor question: Should this comment be a `TODO`?

Reply:
I think 15min works for now, since cot verification takes <1min these days; I was just leaving a note in case we want to modify it in the future.